test/globalize_validations_test.rb

```diff
@@ -165,6 +176,22 @@ class PageWithComposedLocale < ActiveRecord::Base
   test "returns errors for composed locales" do
     page = PageWithComposedLocale.new(title_en: "Title")
     page.valid?
-    assert_equal ["can't be blank"], page.errors[:title_zh_tw]
+    assert_equal ["can't be blank"], page.errors[:title_zh_tw].gsub("`", "'")
+  end
+
+  test "returns only one error when only default locale is invalid and others are provided" do
```

Comment:
New tests do not actually test new scenario, it's just an extra to make sure it does not break existing behaviour in default setting.